src/analyses/variable-sensitivity/variable_sensitivity_configuration.cpp

```diff
@@ -41,9 +41,7 @@ vsd_configt vsd_configt::from_options(const optionst &options)
 
   // This should always be on (for efficeny with 3-way merge)
```

Comment:
I'm ... not 100% sure this is actually true, perhaps we should re-visit this bit of the config.